site/source/docs/getting_started/test-suite.rst

```diff
@@ -96,12 +96,13 @@ Replace ``100`` with another number as you prefer. This will run that number of
 Important Tests
 ===============
 
-When you want to run the entire test suite locally, these are the important commands:
+Please see the bottom the file `tests/test_core.py <https://github.com/emscripten-core/emscripten/blob/master/tests/test_core.py>`_ for the current test modes, as they may change slowly over time.
+When you want to run the entire test suite locally, these are currently the important commands:
 
 .. code-block:: bash
 
   # Run all core asm.js and wasm tests
-  python tests/runner.py asm* binaryen*
+  python tests/runner.py asm* wasm*
```

Comment:
for this example, how about removing the `asm*` (which won't work in the default wasm backend)? please edit the comment to remove the asm.js there too.

Reply:
Done!